### PR TITLE
Config updates for new workgroups and applications

### PR DESCRIPTION
**Pull request by mddilley**
https://github.com/cityofaustin/atd-data-tech/issues/12752

This PR updates the mappings between form field values in the Knack DTS Service Portal app and their corresponding GitHub labels. Both products (applications) and workgroups (divisions) were updated.

The GitHub label additions and edits are not live yet so the best way to test this is to compare the code to the values in the following Google sheets:
- [Workgroups](https://docs.google.com/spreadsheets/d/1BsOklJsJiIIqdlmtNtr9peL3URS7Hdu9RVj4D97701k/edit#gid=968067157)
- [Products](https://docs.google.com/spreadsheets/d/1BsOklJsJiIIqdlmtNtr9peL3URS7Hdu9RVj4D97701k/edit#gid=1546625311)

We want to make sure that:
1. the values in the **New Knack Name** and **New Github Label** columns are used in the code
2. the values marked in red are removed from the code

### CODE REVIEW

File: intake.py
Comment (mddilley) [inline intake.py:1]: All these changes came from [Black code formatter](https://github.com/psf/black)

**Review by johnclary — APPROVED** [review]
looks great—thanks @mddilley!

🚢 🚢 🚢

**Review by frankhereford — APPROVED** [review]
🚀💻📦 Thank you Mike!!

**Review by Charlie-Henry — APPROVED** [review]
Thanks for doing this Mike. We may want to update the `ASSIGNEES` in config.py like putting Andrew down for `gis`, idk what those are used for exactly though.

**Comment by mddilley** [comment]
@Charlie-Henry Thanks for pointing that out! The script assigns all issues to `atdservicebot` now so that config can be removed. I might do that in here, but I'll probably create a backlog issue to limit the number of changes going in to the launch of these updates.

**Comment by johnclary** [comment]
Ah yes: thanks @Charlie-Henry for pointing out cruft i shoulda removed a long time ago. @mddilley i'm for removing it now + yolo merge.

**Comment by mddilley** [comment]
YOLO!!! Thanks, y'all 🙏

**Review by chiaberry — APPROVED** [review]
approving !!

**Pull request merged**